obstacle_stop_planner_refine/include/obstacle_stop_planner/util.hpp

```diff
@@ -0,0 +1,136 @@
+// Copyright 2019 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef OBSTACLE_STOP_PLANNER__UTIL_HPP_
+#define OBSTACLE_STOP_PLANNER__UTIL_HPP_
+
+#include <string>
+
+#include "opencv2/core/core.hpp"
+#include "geometry_msgs/msg/point.hpp"
+#include "boost/geometry.hpp"
+#include "boost/format.hpp"
+#include "boost/assign/list_of.hpp"
+#include "tf2/utils.h"
+#include "diagnostic_msgs/msg/diagnostic_status.hpp"
+#include "diagnostic_msgs/msg/key_value.hpp"
+
+namespace bg = boost::geometry;
+using Point = bg::model::d2::point_xy<double>;
+using Polygon = bg::model::polygon<Point, false>;  // clockwise = false
+using Line = bg::model::linestring<Point>;
+
+namespace
+{
+inline double getYawFromGeometryMsgsQuaternion(const geometry_msgs::msg::Quaternion & quat)
+{
+  tf2::Quaternion tf2_quat(quat.x, quat.y, quat.z, quat.w);
+  double roll, pitch, yaw;
+  tf2::Matrix3x3(tf2_quat).getRPY(roll, pitch, yaw);
+
+  return yaw;
+}
+
+inline std::string jsonDumpsPose(const geometry_msgs::msg::Pose & pose)
+{
+  const std::string json_dumps_pose =
+    (boost::format(
+      R"({"position":{"x":%lf,"y":%lf,"z":%lf},"orientation":{"w":%lf,"x":%lf,"y":%lf,"z":%lf}})") %
+    pose.position.x % pose.position.y % pose.position.z % pose.orientation.w % pose.orientation.x %
+    pose.orientation.y % pose.orientation.z)
+    .str();
+  return json_dumps_pose;
+}
```

Comment:
`jsonDumpsPose` is for an old feature, it can be removed now. :smile:

Reply:
@kenji-miyake This function used here.<https://github.com/tier4/obstacle_stop_planner_refine/pull/3/files#diff-f968bdcfa273891680af99f9ac1520facb3d11da2f8684fc917cd8e15f8cf617R64>. 
How can I resolve it?

Reply:
@KeisukeShima Please remove the related code as well!

Reply:
OK. Thanks!

Reply:
Commit is 622d11e